Documents/How-to-use.md

```diff
@@ -83,7 +83,7 @@ If you add a custom test menu, you will need to re-build the Jenkins menu, and t
                'CustomVHDURL' text box.
             b. Leave 'Kernel' unchanged unless you would like to use customized kernel code or linux-next.
             c. Update the 3 options regarding how to select test cases; TestName, Category and Tag.
-                i. Supported platform: Azure, HyperV, etc.
+                i. Supported platform: Azure, HyperV, WSL, etc.
```

Comment:
would suggest to remove "etc."
This line will have to be updated anyway when a new platform is added, so it's a pretty much static list.